README.md

```diff
@@ -1,65 +1,118 @@
-### How to build: (run from root dir)
+<img height="90" alt="MongoDB Atlas JDBC Driver" align="right" src="resources/media/MongoDBAtlasJDBC.png" />
+
+# MongoDB Atlas JDBC Driver
+
+MongoDB Atlas JDBC provides SQL connectivity to [Atlas](https://www.mongodb.com/atlas) databases and datalakes for client applications developed in Java.
+MongoDB Atlas JDBC is a JDBC Type 4 driver compatible with the JDBC 4.2 specification.
+
+### Prerequisites
+MongoDB Atlas JDBC driver requires Java 1.8 or higher.
+
+## Getting the Driver
+
+### Download and Install
+You can download the precompiled driver (jar) from [Maven Central](https://search.maven.org/artifact/org.mongodb/mongodb-jdbc).
+
+### Build From Source
+To build and test the driver run the following commands from root dir.
+
+#### To build the lean jar which does not include the dependencies
 ```
 ./gradlew clean build
 ```
-### To run demo locally
+
+#### To run demo locally
 ```
 ./gradlew clean :demo:run
 ```
-### To generate source Jar
+
+#### To generate the source Jar
 ```
 ./gradlew clean :sourceJar  
 ```
 You can find the generated jar in build/libs/
-### To generate fat Jar which includes all the dependencies
+
+#### To generate the fat Jar which includes all the dependencies
 ```
 ./gradlew clean :shadowJar
 ```
 You can find the generated jar in build/libs/
-### To generate test Jar
+
+#### To generate the test Jar
 ```
 ./gradlew clean :testJar  
 ```
-You can find the generated jar in build/libs/ 
-### To run the unit tests
+You can find the generated jar in build/libs/
+
+#### To run the unit tests
 ```
 ./gradlew clean test
 ```
-### To fix lint problem
+
+#### To fix lint problem
 ```
 ./gradlew spotlessApply
 ```
-### Release process
-Make sure the following environment variables set:
-- NEXUS_USERNAME
-- NEXUS_PASSWORD
-- NEXUS_PROFILE_ID
-- NEXUS_URL
-- SIGNING_KEY_ID
-- SIGNING_PASSWORD
-- RING_FILE_GPG_BASE64
 
+## Usage
```

Comment:
suggestion: let's move "Prerequisites" and "Download & Install" under this "Usage" heading, and have it be the first thing in the README after the summary, so that users don't have to scroll through any build/development instructions to find the usage info

Reply:
Sounds good to me. Done 👍